model/testutil/stats.go

```diff
@@ -0,0 +1,257 @@
+package testutil
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/apimodels"
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/model/testresult"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type TaskStatus struct {
+	Status         string
+	DetailsType    string
+	DetailsTimeout bool
+	TimeTaken      time.Duration
+}
+
+var Success100 = TaskStatus{"success", "test", false, 100 * 1000 * 1000 * 1000}
+var Success200 = TaskStatus{"success", "test", false, 200 * 1000 * 1000 * 1000}
+var TestFailed = TaskStatus{"failed", "test", false, 20}
+var Timeout = TaskStatus{"failed", "test", true, 100}
+var SetupFailed = TaskStatus{"failed", "setup", false, 10}
+var SystemFailed = TaskStatus{"failed", "system", false, 10}
```

Comment:
I'm super skeptical of three aspects of this: 
- mutable data structures in package level 
- exported mutable data structures. 
- the literal constructor with unkeyed fields.

Reply:
I had moved this from stats_test.go to try and share this code, but it turns out it isn't needed with some of the other changes I made, so I reverted it.